packages/flutter/lib/src/widgets/overlay.dart

```diff
@@ -86,8 +86,7 @@ class OverlayEntry {
     if (_opaque == value)
       return;
     _opaque = value;
-    assert(_overlay != null);
```

Comment:
I think this change make sense, but how didn't we encounter this case prior to this change?

Reply:
We were only modified the opaqueness after the OverlayEntry was added to the Overlay (or when constructing the OverlayEntry by passing the opaque parameter in the constructor).